content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md

```diff
@@ -0,0 +1,133 @@
+---
+title: 'Experience Report: Adopting OpenTelemetry for Metrics in Cloud Foundry'
```

Comment:
How about:

- Using "at" instead of "in" (unless you mean the platform, in which case consider adding "Platform"
- Consider dropping "Experience Report: " from the title and mentioning that in the introductory paragraph. This will make the title a little les wieldy. 

```suggestion
title: Adopting OpenTelemetry for Metrics at Cloud Foundry
```

Reply:
We dropped experience report per your suggestion. We do mean the Cloud Foundry platform, which we usually refer to as just "Cloud Foundry" so I'm thinking we'll leave it that way. "in the Cloud Foundry platform" starts getting a little wordy

Reply:
Sounds good. Thanks for the clarification, and I agree with not adding "platform".

I'm still seeing "Experience Report:" so maybe that edit is on it's way. (If you drop "Experience Report:" you don't need the quotes around the title anymore (FYI).

Reply:
That's right, @ctlong and I went back and forth on dropping it or keeping it in and it is still there for now.

We made an effort in our post to make it useful to anyone thinking about adopting open telemetry for metrics. It seemed like specifying that it's an experience report instead of an announcement might draw in more readers. WDYT?

Reply:
Works for me! Let's keep it 👍🏻